floor/lib/src/sqflite_database_factory.dart

```diff
@@ -1,6 +1,7 @@
 import 'dart:io';
 
 import 'package:path/path.dart';
+// ignore: unnecessary_import
 import 'package:sqflite/sqflite.dart';
```

Comment:
If it's unnecessary, can it not just be removed?

Reply:
The import is necessary. I added the ignore because when we run the build_runner, it indicates that it's an unnecessary import, causing the pipeline to fail.